### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'compat')

### ISSUE THREAD

**Issue by s-pcode** [emotion-js/emotion#2992]
**Current behavior:**

I don't know what is causing this error but I'm not able to see my pages whenever I start my server I get the above error message. I'm attaching a simple 404 page code from my codebase

**To reproduce:**

```
import { motion } from "framer-motion";
import Link from "next/link";
// material
import { styled } from "@mui/material/styles";
import { Box, Button, Typography, Container } from "@mui/material";
// components
import { MotionContainer, varBounceIn } from "../src/components/animate";
import Page from "../src/components/Page";
import { PageNotFoundIllustration } from "../src/assets";

// ----------------------------------------------------------------------

const RootStyle = styled(Page)(({ theme }) => ({
  display: "flex",
  minHeight: "100%",
  alignItems: "center",
  paddingTop: theme.spacing(5),
  paddingBottom: theme.spacing(10),
}));

// ----------------------------------------------------------------------

export default function Page404() {
  return (
    <RootStyle title="404 Page Not Found | aarnâ">
      <Container>
        <MotionContainer initial="initial" open>
          <Box sx={{ maxWidth: 480, margin: "auto", textAlign: "center" }}>
            <motion.div variants={varBounceIn}>
              <Typography variant="h3" paragraph>
                sorry, page not found!
              </Typography>
            </motion.div>
            <Typography sx={{ color: "text.secondary" }}>
              sorry, we couldn't find the page you're looking for. perhaps
              you've mistyped the URL? be sure to check your spelling.
            </Typography>

            <motion.div variants={varBounceIn}>
              <PageNotFoundIllustration
                sx={{ height: 260, my: { xs: 5, sm: 10 } }}
              />
            </motion.div>

            <Link href="/">
              <Button
                size="large"
                variant="contained"
                sx={{ textTransform: "none" }}
              >
                go to home
              </Button>
            </Link>
          </Box>
        </MotionContainer>
      </Container>
    </RootStyle>
  );
}
```

**Expected behavior:**

There is something wrong with the emotion cache which I'm not able to figure out. The next app should start in the local server as expected and render the pages.

**Environment information:**

- `react` version: 17.0.2
- `@emotion/react` version: 11.10.5

**Comment by Andarist** [comment]
Please always try to share a repro case in a runnable form - either by providing a git repository to clone or a codesandbox. OSS maintainers usually can't afford the time to set up the repro, even if exact steps are given.

**Comment by Garajluc** [comment]
I have similar problem with "react": "18.2.0", "next": "13.1.6" and "@emotion/react": "11.10.5", "@emotion/styled": "11.10.5". All of the sudden the build fails on TypeError during the prerendering stage:

```
Error occurred prerendering page "/". Read more: https://nextjs.org/docs/messages/prerender-error
TypeError: Cannot read properties of undefined (reading 'compat')
    at /home/ubuntu/console/node_modules/@emotion/react/dist/emotion-react.cjs.prod.js:85:36
    at /home/ubuntu/console/node_modules/@emotion/react/dist/emotion-element-eeaa326f.cjs.prod.js:27:31
    at Wc (/home/ubuntu/console/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:68:44)
    at Zc (/home/ubuntu/console/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:70:253)
    at Z (/home/ubuntu/console/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:76:89)
    at Zc (/home/ubuntu/console/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:70:481)
    at Z (/home/ubuntu/console/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:76:89)
    at Zc (/home/ubuntu/console/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:70:481)
    at Z (/home/ubuntu/console/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:76:89)
    at $c (/home/ubuntu/console/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:78:98)
 ```

**Comment by Andarist** [comment]
It's likely a bundling issue - the call stack shows that a React browser build "calls into" our CJS build. I can't address this without having a repro case for this though, please create one and I will dive into investigating this.

**Comment by santhosh-c1** [comment]
JFYI There also seems to be an approved PR out for this: https://github.com/mui/material-ui/pull/36162

**Comment by s-pcode** [comment]
This was an issue from mui side and they have fixed this in v5.11.9. Please install the new version which works as expected with all NextJS versions. 👍
